basic-api.yml

```diff
@@ -0,0 +1,142 @@
+      "lprCategory": "C09",
+      "lprNumber": "999-999-999",
+      "commuterClassification": "C1",
+      "birthCountry": "Bahamas",
+      "birthDate": "1958-07-17"
+    }
+  }],
+  "proof": [{
+    "type": "Ed25519Signature2018",
+    "created": "2020-02-25T22:03:05Z",
+    "jws": "eyJhbGciOiJFZERTQSIsImI2..hS5SPmWTFocjcDg",
+    "proofPurpose":"assertionMethod",
+    "verificationMethod":"did:key:z6Mkm8aUytv...nvKQrHB"
+  }]
+}
+        '400':
+          description: Malformed request from client.
+components:
+  schemas:
+    IssueRequest:
+      type: object
+      properties:
+        credential:
+          type: object
+          description: The credential to be issued minus the proofs
+          example: {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/citizenship/v1"
+  ],
+  "id": "https://issuer.oidp.uscis.gov/credentials/83627465",
+  "type": [
+    "VerifiableCredential",
+    "PermanentResidentCard"
+  ],
+  "issuanceDate": "2019-12-03T12:19:52Z",
+  "expirationDate": "2029-12-03T12:19:52Z",
+  "credentialSubject": {
+    "id": "did:example:b34ca6cd37bbf23",
+    "type": [
+      "PermanentResident",
+      "Person"
+    ],
+    "givenName": "JOHN",
+    "familyName": "SMITH",
+    "gender": "Male",
+    "image": "data:image/png;base64,iVBORw0KGgo...kJggg==",
+    "residentSince": "2015-01-01",
+    "lprCategory": "C09",
+    "lprNumber": "999-999-999",
+    "commuterClassification": "C1",
+    "birthCountry": "Bahamas",
+    "birthDate": "1958-07-17"
+  }
+}
+        options:
+          type: object
+          description: Optional values that check or modify the input and modify the output of the request.
+          properties:
+            validation:
+              type: object
+              description: Optional validation schemas to use on the input document.
+              properties:
+                type:
+                  type: string
+                  description: The type of validation mechanism
+                  example: JsonSchema2018
+                credentialSchema:
+                  type: string
+                  description: A link to a credential schema to use for validation.
+                  example: https://example.com/schemas/prc.json
+            issuer:
+              type: string
+              description: Optional identifier for the issuer of the credential. If this is missing, it may be pre-configured or implied by other values in the request.
+              example: did:example:ec712ebc6f1c221ebfeb1f76e12
+            output:
+              type: object
+              description: Optional values that modify the output of the request.
+              properties:
+                representation:
+                  description: Optional value to specify the credential representation format. Well-known values are "jsonld" and "json". Other values MAY be supported. If this option is not specified a default of "jsonld" MUST be assumed.
```

Comment:
If this is missing, then the concrete implementation could have a default, or may make the default configurable. I don't think the API itself should have a built-in preference for a specific credential format.

Reply:
AFAIK, the SVIP cohort is only using jsonld credentials, I'd be in favor of making that the default, or making this a required field.

Reply:
Okay I don't really feel strongly about this, but I want to point out that this is now a CCG work item. This should serve broader interests than just what the SVIP cohort needs.

Reply:
> Okay I don't really feel strongly about this, but I want to point out that this is now a CCG work item. This should serve broader interests than just what the SVIP cohort needs.

Agreed *and* part of the standards process is setting sensible defaults if we can. JSON-LD representation *for Verifiable Credentials* is a safe bet given that the spec requires `@context` in both cases. That said, if people object, the fallback should be to make this a required field... which would also mean making the proof format a required field.

Reply:
+1 to JSON-LD as a default for cred format.